src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java

```diff
@@ -96,7 +96,7 @@ public String getNormalizedColumnHeader() {
     */
     @JsonProperty("normalized_column_header")
```

Comment:
This should be changed to "column_header"

Reply:
@averyniceday The column header in the google sheet will still be "normalized_column_header" so I think this is how you get it to read "normalized_column_header" but then write out "column_header".  It seemed to work.